js/src/rive.ts

```diff
@@ -1647,6 +1652,21 @@ export class Rive {
     }
   }
 
+  /**
+   * Enables frames-per-second (FPS) reporting for the runtime
+   * @param fpsCallback - Callback from the runtime during the RAF loop that supplies the FPS value
```

Comment:
// If null, Rive will append a fixed-position div at the top-right corner of the page with the FPS reading.?